docs/multi-stage-query/reference.md

```diff
@@ -602,6 +602,7 @@ The following table lists the context parameters for the MSQ task engine:
 | `faultTolerance` | SELECT, INSERT, REPLACE<br /><br /> Whether to turn on fault tolerance mode or not. Failed workers are retried based on [Limits](#limits). Cannot be used when `durableShuffleStorage` is explicitly set to false.  | `false` |
 | `composedIntermediateSuperSorterStorageEnabled` | SELECT, INSERT, REPLACE<br /><br /> Whether to enable automatic fallback to durable storage from local storage for sorting's intermediate data. Requires to setup `intermediateSuperSorterStorageMaxLocalBytes` limit for local storage and durable shuffle storage feature as well.| `false` |
 | `intermediateSuperSorterStorageMaxLocalBytes` | SELECT, INSERT, REPLACE<br /><br /> Whether to enable a byte limit on local storage for sorting's intermediate data. If that limit is crossed, the task fails with `ResourceLimitExceededException`.| `9223372036854775807` |
+| `maxInputBytesPerWorker` | Maximum number of input bytes per worker, used when assigning input slices to tasks. Used only in case number of tasks is determined automatically. | `10 GB` |
```

Comment:
Can we improve this? What does this limit? Us casual readers can't figure out what "input bytes" means. Input files? Input frames? Something else?

Is this a limit on total input bytes? Probably not. On buffers used to store a chunk of input? On the length of the longest input row? What problem occurs if this is too small?

Why would I want to reduce this? 10 GB is pretty big. Is this a memory limit? Seems big. Or, is it the largest input frame aggregate size that the worker will accept? As you can see, I'm guessing: please explain the actual meaning.

Reply:
+1 on this.